nestkernel/event.h

```diff
@@ -952,6 +952,12 @@ class GapJunctionEvent : public SecondaryEvent
     supported_syn_ids_.push_back( synid );
   }
 
+  static void
+  set_coeff_length( const size_t coeff_length )
+  {
+    coeff_length_ = coeff_length;
```

Comment:
You are right! I tested it by calling the function from a thread-parallel region, it works. I also added this assert to the other two functions from `GapJunctionEvent` that modify static variables